Add filterbuilder component (#904)
enaqx/awesome-react: README.md

* [material-ui-layout - A declarative Layout component based on Material-UI and Google's Material Design](https://github.com/OrigenStudio/material-ui-layout)
* [react-admin](https://github.com/marmelab/react-admin): A frontend Framework for building admin applications running in the browser on top of REST/GraphQL APIs, using ES6, React and Material Design
* [reactjs-popup - popup, Modals, Tooltips and Menus,  All in one](https://github.com/yjose/reactjs-popup)

> * [react-filter-control - The React filterbuilder component for building the filter criteria in the UI](https://github.com/komarovalexander/react-filter-control)

Reviewer: ```suggestion
* [react-filter-control - A component to build UI for boolean filter creation](https://github.com/komarovalexander/react-filter-control)
```
